pallets/author-inherent/src/lib.rs

```diff
@@ -218,154 +218,3 @@ impl InherentError {
 		}
 	}
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Why did these dissapear?

Reply:
Removed in https://github.com/PureStake/nimbus/pull/37 after discussion with team.

tl;dr they were broken/uncompilable and the system under test didn't even exist in code.

Reply:
I see, sorry wasnt aware!